src/mohagames/lockmystuff/Main.php

```diff
@@ -309,11 +322,18 @@ public function isLockedDown(Position $position, Item $item) : ?bool{
             $y = (int) $loc_array[1];
             $z = (int) $loc_array[2];
             if (($item_x == $x && $item_z == $z) && (abs($item_y - $y) <= 1 || abs($y - $item_y) <= 1) && $position->getLevel()->getName() == $row["world"]){
-                    if($item->getCustomName() != $row["door_name"] || $item->getId() != $this->itemID){
+
+
+                    if($item->getCustomName() != $row["door_name"] || $item->getId() != $this->itemID || $item->getLore() != array("Key: ".$row["door_name"])){
                         //If the door exists in the database but it's locked, then "true" will be returned.
                         $check = true;
                         break;
+                    } else if($item->getCustomName() != $row["door_name"] || $item->getId() != $this->itemID){
```

Comment:
Dit werkt niet als verwacht, deze statement wordt de hele tijd getriggered wat vreemd is. Wanneer ik geen key heb dan zegt zegt de plugin dat ik een oude key heb. Ik zal er later nog eens naar kijken